### DISCUSSION TITLE
Can I check 200 websites?

### DISCUSSION THREAD

**Discussion by linkspreed** [upptime/upptime#796] [Q&A] [answered]
Dear Upptime community,

I wondered if it#s possible to check ~200 different websites in one Repo, or is it too much for the Github server?

Greetings

**Comment by AnandChowdhary** [comment, marked as answer]
Yes, I've seen people use Upptime for 100s of websites.

**Reply by linkspreed** [reply]
Great, can you check out: https://github.com/linkspreed/linkspreed_uptime.
The last websites are always marked as offline but they're all online.
Why?

**Reply by AnandChowdhary** [reply]
<img width="844" alt="CleanShot 2023-08-28 at 17 23 50@2x" src="https://github.com/upptime/upptime/assets/2841780/730364c3-a01f-4950-9f38-858ea784c7d6">

You're right, it seems like we're hitting the GitHub API rate limit as we keep updating the data for so many sites. Would be happy to accept a PR to add a delay based on number of sites to check!

**Comment by markgarrigan** [comment]
What about 1000s?

**Reply by AnandChowdhary** [reply]
I've seen people use many sites, but AFAIK nobody has used thousands. I think we'll definitely hit the GitHub rate limit so we'll need some sort of mechanism to add a delay. If you don't care about the GitHub Actions minutes usage, it should work in that case! :)

**Reply by markgarrigan** [reply]
We currently have a little over 1400 sites. I don't care about actions minutes as I'd be using self hosted runners. However, I would care about triggering any rate limits.
